README.md

```diff
@@ -37,15 +37,6 @@ from rust repo, then you should have [MSYS2](http://msys2.github.io/) with
 `mingw-w64-openssl` installed as part of `mingw-w64-x86_64-toolchain`
 (or `mingw-w64-i686-toolchain`) package.
 
-Alternatively, install OpenSSL from [here][1]. Cargo will not be able to find OpenSSL if it's
-installed to the default location. You can either copy the `include/openssl`
-directory, `libssl32.dll`, and `libeay32.dll` to locations that Cargo can find
-or pass the location to Cargo via environment variables:
-
-```bash
-env OPENSSL_LIB_DIR=C:/OpenSSL-Win64 OPENSSL_INCLUDE_DIR=C:/OpenSSL-Win64/include cargo build
```

Comment:
The alternate Windows method only works if MinGW-w64 is already installed, due to the dependency on [gcc-rs](https://github.com/alexcrichton/gcc-rs) (which, in turn, depends on MinGW's version of gcc), so it should not be considered an alternative to using MinGW-w64.

However, when I authored the pull, I was unaware of gcc-rs's dependency on MinGW-w64, and thought the failed compile was an actual bug. The pull would have been a band-aid until the bug was fixed.

This pull may be slightly confused, but the README should reflect that rust-openssl cannot be built without MinGW.